pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/ClinicalOrderMigrator.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsm.model.elastic.migration;
+
+import java.util.Map;
+
+import org.broadinstitute.dsm.db.dao.mercury.ClinicalOrderDao;
+import org.broadinstitute.dsm.statics.ESObjectConstants;
+
+public class ClinicalOrderMigrator extends BaseCollectionMigrator {
```

Comment:
Could you add a top-line comment to this class?  Or better yet, add one to BaseCollectionMigrator BaseMigrator explaining what migrators are built to do?